spanner/spanner_postgresql_jsonb_query_parameter.rb

```diff
@@ -0,0 +1,41 @@
+# Copyright 2022 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START spanner_postgresql_jsonb_query_parameter]
+require "google/cloud/spanner"
+
+def spanner_postgresql_jsonb_query_parameter project_id:, instance_id:, database_id:
+  # project_id  = "Your Google Cloud project ID"
+  # instance_id = "Your Spanner instance ID"
+  # database_id = "Your Spanner database ID"
+
+  sql_query = <<~QUERY
+    SELECT venueid, venuedetails 
+    FROM Venues
+    WHERE CAST(venuedetails ->> 'rating' AS INTEGER) > $1
+  QUERY
+
+  results = client.execute sql_query, params: { p1: 5 }
```

Comment:
What does the `param` mean here? 
I think it'll be good if we add some comments here.

Reply:
Param here is the key value pair for parameterised query's parameter.
Sure will add a comment here 👍